controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/AthenzFacade.java

```diff
@@ -192,6 +193,10 @@ public boolean hasHostedOperatorAccess(AthenzIdentity identity) {
         return hasAccess("modify", service.getDomain().getName() + ":hosted-vespa", identity);
     }
 
+    public boolean canLaunch(AthenzIdentity athenzIdentity, AthenzService service) {
```

Comment:
It's a bit confusing to understand who is launching who from the names of the parameters.